Account/Quotas/README.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
The main Readme file should be updated to to list the new sample.
https://github.com/Azure-Samples/media-services-v3-dotnet/blob/track2/README.md